### PR TITLE
Remove useless published date in resource model

### PR DESCRIPTION
**Pull request by quaxsze**
fix https://github.com/etalab/data.gouv.fr/issues/1033

### CODE REVIEW

**Review by nicolaskempf57 — APPROVED** [review]

File: udata_front/theme/gouvfr/templates/dataset/resource/card.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {% set available = resource.extras['check:status']
     and resource.extras['check:status'] >= 200 and resource.extras['check:status'] < 400 %}
 {% set unavailable = resource.extras['check:status'] and resource.extras['check:status'] >= 400 %}
-{% set last_update = resource.published if resource.published > resource.modified else resource.modified %}
+{% set last_update = resource.modified %}
```
Comment (maudetes) [inline udata_front/theme/gouvfr/templates/dataset/resource/card.html:6]: Same comment: We should take harvest into account (may be done in https://github.com/etalab/udata-front/pull/201)

Comment (quaxsze) [reply udata_front/theme/gouvfr/templates/dataset/resource/card.html:6]: done in [#201](https://github.com/etalab/udata-front/pull/201)

File: theme/js/components/dataset/resource/resource.vue [hunk 2]
```diff
@@ -259,7 +259,7 @@ export default defineComponent({
       }
     }
     const availabilityChecked = computed(() => props.resource.extras && props.resource.extras['check:status']);
-    const lastUpdate = computed(() => props.resource.published > props.resource.last_modified ? props.resource.published : props.resource.last_modified);
+    const lastUpdate = computed(() => props.resource.last_modified);
```
[the remark this answers is not in the document]
Comment (quaxsze) [reply theme/js/components/dataset/resource/resource.vue:262]: done in [#201](https://github.com/etalab/udata-front/pull/201)

**Review by maudetes — APPROVED** [review]
:ok_hand:

**Pull request merged**